javagen/src/main/java/com/azure/autorest/template/ServiceVersionTemplate.java

```diff
@@ -22,19 +20,14 @@ public static ServiceVersionTemplate getInstance() {
 
     @Override
     public void write(ServiceVersion serviceVersion, JavaFile javaFile) {
-        // imports
-        Set<String> imports = new HashSet<>();
-        imports.add("com.azure.core.util.ServiceVersion");
-        javaFile.declareImport(imports);
-
         javaFile.javadocComment(comment -> {
             comment.description("Service version of " + serviceVersion.getServiceName());
         });
 
         String className = serviceVersion.getClassName();
         List<String> serviceVersions = serviceVersion.getServiceVersions();
 
-        javaFile.publicEnum(className + " implements ServiceVersion", classBlock -> {
+        javaFile.publicEnum(className + " implements com.azure.core.util.ServiceVersion", classBlock -> {
```

Comment:
We should still import the ServiceVersion enum and not have the fully qualified name here in `implements` section.